project_status/models/project.py

```diff
@@ -0,0 +1,18 @@
+from odoo import SUPERUSER_ID, api, fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    @api.model
+    def _read_group_status_ids(self, statuses, domain, order):
+        statuse_ids = statuses._search([], order=order, access_rights_uid=SUPERUSER_ID)
```

Comment:
```suggestion
        status_ids = statuses._search([], order=order, access_rights_uid=SUPERUSER_ID)
```

i'd rename that one.

Reply:
Fine with me.

Reply:
Hello @igallart do you want to include this before it's merged?

Reply:
Hello @leemannd , I don't have this module very fresh, I just tried to migrate it to make it functional, so I don't know how it works 100%. If you think that code is not necessary, you are free to remove it, I can't help you much more.